apps/modernization-ui/src/design-system/alert/AlertMessage.spec.tsx

```diff
@@ -106,4 +119,15 @@ describe('AlertMessage', () => {
 
         expect(getByRole('alert', { name: 'The title. Custom aria label' })).toBeInTheDocument();
     });
+
+    it('should close the alert when the close button is clicked', () => {
+        render(<AlertWrapper />);
+
+        expect(screen.getByRole('alert')).toBeInTheDocument();
+
+        const closeButton = screen.getByRole('button', { name: /close alert/i });
+        fireEvent.click(closeButton);
```

Comment:
Suggestion: It is recommended to use `userEvent` to invoke actions a user would perform.

```typescript
const user = userEvent.setup();

render(<AlertWrapper />);

...

await user.click(closeButton);